sprints/sprint_3.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
```

Comment:
neet both en and ru versions. you can use the previous one as an example basically

Reply:
Done, and also added appropriate links, thx